client/src/pages/Resource/index.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { ResourceService, useService } from "../../services";
+
+import "./Resource.scss";
+
+export default function Resource() {
+	const [resource, setResource] = useState(undefined);
+	const resourceService = useService(ResourceService);
+	const { id } = useParams();
+
+	useEffect(() => {
```

Comment:
That's true, but it's more consistent with other similar uses of the services to use `.then`. You can use `.catch` to do the same thing `try`/`catch` does, but there's not (in my opinion) a huge benefit to explicitly writing an error message to the console rather than letting the browser write the default one.